### ISSUE TITLE
xpra initialization error: mode 'seamless' not found in command line arguments

### ISSUE THREAD

**Issue by badmin-c** [Xpra-org/xpra#3679]
We are using the following client command to start a google-chome session:
```shell
xpra.exe start --ssh=plink ssh:user@xpra1.nmc.netcologne.intern \
    --log-dir=~ --html=off --printing=no --server-idle-timeout=300 --env=LANG=de_DE.UTF-8 \
    --exit-with-children=yes --exit-with-client=yes --exit-with-windows=yes \
    --start-child="google-chrome --proxy-pac-url=http://proxy.location.domain.tld/proxy.pac"
```
It works fine on most clients, but on a few the session terminates immediately after showing google-chrome for like 1-3 seconds. A popup windows then states: xpra initialization error: mode 'seamless' not found in command line arguments

Server OS: Debian 10
xpra Server Version : 4.4-r0-1

Client: Windows 10, latest Updates
xpra Client Version: 4.4.1

I tried to reproduce the behaviour on another machine, to no avail. Reinstalling the Client did not help. 

server.log:
```
2022-11-04 12:14:05,802 no uinput module (not usually needed)
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running
2022-11-04 12:14:07,213 pointer device emulation using XTest
2022-11-04 12:14:07,236 wrote pid 2809 to '/run/user/2029/xpra/3/server.pid'
2022-11-04 12:14:07,257 created unix domain socket '/run/user/2029/xpra/xpra1-3'
2022-11-04 12:14:07,258 cannot create group socket '/run/xpra/xpra1-3'
2022-11-04 12:14:07,258  [Errno 13] Keine Berechtigung
2022-11-04 12:14:07,260 created unix domain socket '/home/user/.xpra/xpra1-3'
2022-11-04 12:14:07,262 created unix domain socket '/run/user/2029/xpra/3/socket'
2022-11-04 12:14:07,262 xvfb pid=2815
2022-11-04 12:14:07,313 D-Bus notification forwarding is available
2022-11-04 12:14:07,321 pulseaudio not started: 'pulseaudio' command not found
2022-11-04 12:14:07,322 removing private directory '/run/user/2029/xpra/3/pulse'
2022-11-04 12:14:07,376 watching for applications menu changes in:
2022-11-04 12:14:07,376  '/usr/share/applications'
2022-11-04 12:14:07,386 Warning: webcam forwarding is disabled
2022-11-04 12:14:07,388  the virtual video directory '/sys/devices/virtual/video4linux' was not found
2022-11-04 12:14:07,388  make sure that the 'v4l2loopback' kernel module is installed and loaded
2022-11-04 12:14:07,388  or use the 'webcam=no' option
2022-11-04 12:14:07,388 found 0 virtual video devices for webcam forwarding
2022-11-04 12:14:07,398 started command `google-chrome --proxy-pac-url=http://proxy.location.domain.tld/proxy.pac` with pid 2859
2022-11-04 12:14:07,445 Error parsing xdg menu data:
2022-11-04 12:14:07,447  ParsingError in file '/etc/xdg/menus/kde-debian-menu.menu', File not found
2022-11-04 12:14:07,448  this is either a bug in python-xdg,
2022-11-04 12:14:07,448  or an invalid system menu configuration
2022-11-04 12:14:07,507 31.4GB of system memory
2022-11-04 12:14:07,598 GStreamer loaded
2022-11-04 12:14:07,599 waiting for initialization thread to complete
2022-11-04 12:14:07,600 xpra is ready.
libva error: va_getDriverName() failed with unknown libva error,driver_name=(null)
[2912:2912:1104/121407.668987:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
2022-11-04 12:14:07,780 New unix-domain connection received
2022-11-04 12:14:07,780  on '/run/user/2029/xpra/xpra1-3'
[2859:2978:1104/121407.787385:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.Properties.Get: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[2859:2978:1104/121407.787849:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.UPower.GetDisplayDevice: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[2859:2978:1104/121407.788181:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.UPower.EnumerateDevices: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
2022-11-04 12:14:07,904 New unix-domain connection received
2022-11-04 12:14:07,905  on '/run/user/2029/xpra/xpra1-3'
2022-11-04 12:14:07,909 Handshake complete; enabling connection
2022-11-04 12:14:07,925 Error: failed to allocate a CUDA context:
2022-11-04 12:14:07,925  No module named 'pycuda'
2022-11-04 12:14:07,926  NVJPEG and NVENC will not be available
2022-11-04 12:14:07,926 Python/GTK3 Microsoft Windows 10 aero client version 4.4
2022-11-04 12:14:07,926  connected from 'unknown host' as 'user'
2022-11-04 12:14:07,926  OpenGL is enabled with Intel(R) HD Graphics 530
2022-11-04 12:14:07,927 setting key repeat rate from client: 500ms delay / 33ms interval
2022-11-04 12:14:07,928 setting keyboard layout to 'de'
2022-11-04 12:14:08,006  client root window size is 3840x1080
2022-11-04 12:14:08,006   Default (1016x285 mm - DPI: 96x96)
2022-11-04 12:14:08,006     (Standardmonitortypen) TIO24Gen3 1920x1080 at    0x0    (527x296 mm - DPI: 93x93) workarea: 1920x1040
2022-11-04 12:14:08,006     (Standardmonitortypen) LEN T24i-10 1920x1080 at 1920x0    (527x296 mm - DPI: 93x93) workarea: 1920x1080 at 1920x0
2022-11-04 12:14:08,213 OpenGL is supported on display ':3'
2022-11-04 12:14:08,213  using 'llvmpipe (LLVM 7.0, 256 bits)' renderer
2022-11-04 12:14:08,514 server virtual display now set to 3840x1080
2022-11-04 12:14:08,522  automatic picture encoding enabled, also available:
2022-11-04 12:14:08,522   h264, vp9, vp8, png, png/P, png/L, webp, rgb24, rgb32, jpeg, jpega, scroll
2022-11-04 12:14:08,526 Error parsing xdg menu data:
2022-11-04 12:14:08,527  ParsingError in file '/etc/xdg/menus/kde-debian-menu.menu', File not found
2022-11-04 12:14:08,529  this is either a bug in python-xdg,
2022-11-04 12:14:08,529  or an invalid system menu configuration
2022-11-04 12:14:08,540 DPI set to 96 x 96
2022-11-04 12:14:08,572 client   1 @11.391 Xpra X11 seamless server version 4.4
2022-11-04 12:14:08,575 client   1 @11.391  running on unknown
2022-11-04 12:14:08,906 xpra X11 seamless version 4.4-r0 64-bit
2022-11-04 12:14:08,906  uid=2029 (user), gid=2004 (sdpccm)
2022-11-04 12:14:08,908  running with pid 2809 on Linux Debian 10 buster
2022-11-04 12:14:08,909  connected to X11 display :3 with 24 bit colors
2022-11-04 12:14:09,597 client   1 @12.406 running, 1 window
2022-11-04 12:14:09,610 Error parsing xdg menu data:
2022-11-04 12:14:09,611  ParsingError in file '/etc/xdg/menus/kde-debian-menu.menu', File not found
2022-11-04 12:14:09,611  this is either a bug in python-xdg,
2022-11-04 12:14:09,611  or an invalid system menu configuration

(Xpra:2809): Gdk-CRITICAL **: 12:14:09.643: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
2022-11-04 12:14:09,864 Error: failed to encode packet: [38, 1, 944, 0, 1, 1020, 'rgb24', <memory at 0x7efe201bba08>, 3, 4, {'rgb_format': 'BGRX', 'lz4': 2, 'flush': 1, 'window-size': (945, 1020)}]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 649, in encode
    main_packet, proto_flags = self._encoder(packet)
  File "/usr/lib/python3/dist-packages/xpra/net/packet_encoding.py", line 36, in do_rencode
    return rencode_dumps(v), FLAGS_RENCODE
  File "rencode/rencode.pyx", line 334, in rencode._rencode.dumps
  File "rencode/rencode.pyx", line 314, in rencode._rencode.encode
  File "rencode/rencode.pyx", line 247, in rencode._rencode.encode_list
  File "rencode/rencode.pyx", line 320, in rencode._rencode.encode
Exception: type <class 'memoryview'> not handled
2022-11-04 12:14:09,864 unsupported type: <class 'memoryview'> in 'draw' packet->[7]
2022-11-04 12:14:09,864 Error: error in network packet write/format
2022-11-04 12:14:09,864  type <class 'memoryview'> not handled
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 404, in write_format_thread_loop
    self._add_packet_to_queue(*gpc())
  File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 421, in _add_packet_to_queue
    chunks = self.encode(packet)
  File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 649, in encode
    main_packet, proto_flags = self._encoder(packet)
  File "/usr/lib/python3/dist-packages/xpra/net/packet_encoding.py", line 36, in do_rencode
    return rencode_dumps(v), FLAGS_RENCODE
  File "rencode/rencode.pyx", line 334, in rencode._rencode.dumps
  File "rencode/rencode.pyx", line 314, in rencode._rencode.encode
  File "rencode/rencode.pyx", line 247, in rencode._rencode.encode_list
  File "rencode/rencode.pyx", line 320, in rencode._rencode.encode
Exception: type <class 'memoryview'> not handled
2022-11-04 12:14:09,866 Python/GTK3 client 1 disconnected.
2022-11-04 12:14:09,867 Last client has disconnected, terminating
2022-11-04 12:14:09,867 xpra X11 seamless server is terminating
2022-11-04 12:14:09,885 removing unix domain socket '/run/user/2029/xpra/xpra1-3'
2022-11-04 12:14:09,886 removing unix domain socket '/home/user/.xpra/xpra1-3'
2022-11-04 12:14:09,886 removing unix domain socket '/run/user/2029/xpra/3/socket'
[2859:2896:1104/121410.187799:FATAL:bus.cc(1226)] D-Bus connection was disconnected. Aborting.
2022-11-04 12:14:10,188 killing xvfb with pid 2815
Gdk-Message: 12:14:10.190: Xpra: Fatal IO error 0 (Erfolg) on X server :3.

[2912:2912:1104/121410.188418:ERROR:connection.cc(46)] X connection error received.
[2912:2936:1104/121410.189291:ERROR:connection.cc(46)] X connection error received.
[1104/121410.199775:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq: No such file or directory (2)
[1104/121410.199820:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq: No such file or directory (2)
Gdk-Message: 12:14:10.203: google-chrome: Fatal IO error 11 (Die Ressource ist zur Zeit nicht verfügbar) on X server :3.

[1104/121410.203649:ERROR:exception_handler_server.cc(131)] tgkill: No such process (3)
```

**Comment by totaam** [comment]
Your error log shows:
```
Error: failed to encode packet: [38, 1, 944, 0, 1, 1020, 'rgb24', <memory at 0x7efe201bba08>, 3, 4, {'rgb_format': 'BGRX', 'lz4': 2, 'flush': 1, 'window-size': (945, 1020)}]
```

And in 4.4 we have: bb773413a6a4d9b41a97d5e4eee739c5c61e9f2e which is specifically there for legacy packet encoders:
https://github.com/Xpra-org/xpra/blob/df57f3728c393dd381ac26e0208e770e0167eae4/xpra/net/protocol.py#L594-L597

@badmin-c Are you certain that your server is now running 4.4.1?
Try `xpra version ssh://user@xpra1.nmc.netcologne.intern/`

**Comment by badmin-c** [comment]
xpra version ssh://user@xpra1.nmc.netcologne.intern/

```ssh server OS is 'linux-gnu'
 SSH: 'xpra initialization error:'
 SSH: ' cannot find any live servers to connect to'
SSH EOF on stderr of run-xpra
Connection failed
```
To me it looks like the right version. On the server:
```
root@xpra1:~# xpra --version
xpra v4.4.1-r0
```
Client Version: (.zip package)
```
Xpra-Python3-x86_64_4.4.1-r0
```

server.log:
```
server.log:
2022-11-04 15:57:43,688 no uinput module (not usually needed)
2022-11-04 15:57:45,530 pointer device emulation using XTest
2022-11-04 15:57:45,573 wrote pid 5694 to '/run/user/2029/xpra/0/server.pid'
2022-11-04 15:57:45,608 created unix domain socket '/run/user/2029/xpra/xpra1-0'
2022-11-04 15:57:45,608 cannot create group socket '/run/xpra/xpra1-0'
2022-11-04 15:57:45,608 [Errno 13] Keine Berechtigung
2022-11-04 15:57:45,610 created unix domain socket '/home/user/.xpra/xpra1-0'
2022-11-04 15:57:45,612 created unix domain socket '/run/user/2029/xpra/0/socket'
2022-11-04 15:57:45,612 xvfb pid=5700
2022-11-04 15:57:45,655 D-Bus notification forwarding is available
2022-11-04 15:57:45,662 pulseaudio not started: 'pulseaudio' command not found
2022-11-04 15:57:45,662 removing private directory '/run/user/2029/xpra/0/pulse'
2022-11-04 15:57:45,714 watching for applications menu changes in:
2022-11-04 15:57:45,714 '/usr/share/applications'
2022-11-04 15:57:45,729 Error parsing xdg menu data:
2022-11-04 15:57:45,729 ParsingError in file '/etc/xdg/menus/kde-debian-menu.menu', File not found
2022-11-04 15:57:45,729 this is either a bug in python-xdg,
2022-11-04 15:57:45,730 or an invalid system menu configuration
2022-11-04 15:57:45,798 Warning: webcam forwarding is disabled
2022-11-04 15:57:45,798 the virtual video directory '/sys/devices/virtual/video4linux' was not found
2022-11-04 15:57:45,798 make sure that the 'v4l2loopback' kernel module is installed and loaded
2022-11-04 15:57:45,798 or use the 'webcam=no' option
2022-11-04 15:57:45,798 found 0 virtual video devices for webcam forwarding
2022-11-04 15:57:45,808 started command google-chrome --proxy-pac-url=http://proxy.location.domain.tld/proxy.pac with pid 5745
2022-11-04 15:57:45,874 GStreamer version 1.14.4
2022-11-04 15:57:46,239 waiting for initialization thread to complete
2022-11-04 15:57:46,239 xpra is ready.
2022-11-04 15:57:46,255 31.4GB of system memory
2022-11-04 15:57:46,410 New unix-domain connection received
2022-11-04 15:57:46,411 on '/home/user/.xpra/xpra1-0'
2022-11-04 15:57:46,506 New unix-domain connection received
2022-11-04 15:57:46,506 on '/home/user/.xpra/xpra1-0'
2022-11-04 15:57:46,508 Handshake complete; enabling connection
2022-11-04 15:57:46,533 Error: failed to allocate a CUDA context:
2022-11-04 15:57:46,533 No module named 'pycuda'
2022-11-04 15:57:46,533 NVJPEG and NVENC will not be available
2022-11-04 15:57:46,533 Python/GTK3 Microsoft Windows 10 aero client version 4.4
2022-11-04 15:57:46,533 connected from 'unknown host' as 'user'
2022-11-04 15:57:46,533 OpenGL is enabled with Intel(R) HD Graphics 530
2022-11-04 15:57:46,537 setting key repeat rate from client: 500ms delay / 33ms interval
2022-11-04 15:57:46,540 setting keyboard layout to 'de'
2022-11-04 15:57:46,593 client root window size is 3840x1080
2022-11-04 15:57:46,594 Default (1016x285 mm - DPI: 96x96)
2022-11-04 15:57:46,594 (Standardmonitortypen) TIO24Gen3 1920x1080 at 0x0 (527x296 mm - DPI: 93x93) workarea: 1920x1040
2022-11-04 15:57:46,594 (Standardmonitortypen) LEN T24i-10 1920x1080 at 1920x0 (527x296 mm - DPI: 93x93) workarea: 1920x1080 at 1920x0
libva error: va_getDriverName() failed with unknown libva error,driver_name=(null)
[5826:5826:1104/155746.887071:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
2022-11-04 15:57:47,096 server virtual display now set to 3840x1080
2022-11-04 15:57:47,103 Error parsing xdg menu data:
2022-11-04 15:57:47,103 ParsingError in file '/etc/xdg/menus/kde-debian-menu.menu', File not found
2022-11-04 15:57:47,103 this is either a bug in python-xdg,
2022-11-04 15:57:47,103 or an invalid system menu configuration
2022-11-04 15:57:47,107 DPI set to 96 x 96
2022-11-04 15:57:47,145 client 1 @13.875 Xpra X11 seamless server version 4.4
2022-11-04 15:57:47,147 client 1 @13.875 running on unknown
2022-11-04 15:57:47,193 client 1 @13.922 running
[5745:5963:1104/155747.226754:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.Properties.Get: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[5745:5963:1104/155747.226935:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.UPower.GetDisplayDevice: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
[5745:5963:1104/155747.227018:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.UPower.EnumerateDevices: object_path= /org/freedesktop/UPower: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.UPower was not provided by any .service files
2022-11-04 15:57:47,235 Error parsing xdg menu data:
2022-11-04 15:57:47,235 ParsingError in file '/etc/xdg/menus/kde-debian-menu.menu', File not found
2022-11-04 15:57:47,235 this is either a bug in python-xdg,
2022-11-04 15:57:47,235 or an invalid system menu configuration
[5876:25:1104/155747.238839:ERROR:command_buffer_proxy_impl.cc(128)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
2022-11-04 15:57:47,336 xpra X11 seamless version 4.4.1-r0 64-bit
2022-11-04 15:57:47,336 uid=2029 (user), gid=2004 (sdpccm)
2022-11-04 15:57:47,336 running with pid 5694 on Linux Debian 10 buster
2022-11-04 15:57:47,344 connected to X11 display :0 with 24 bit colors
2022-11-04 15:57:47,401 Error: failed to encode packet: [38, 1, 944, 0, 1, 1020, 'rgb24', <memory at 0x7f370021f888>, 4, 4, {'rgb_format': 'BGRX', 'lz4': 1, 'flush': 1, 'window-size': (945, 1020)}]
Traceback (most recent call last):
File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 649, in encode
main_packet, proto_flags = self._encoder(packet)
File "/usr/lib/python3/dist-packages/xpra/net/packet_encoding.py", line 36, in do_rencode
return rencode_dumps(v), FLAGS_RENCODE
File "rencode/rencode.pyx", line 334, in rencode._rencode.dumps
File "rencode/rencode.pyx", line 314, in rencode._rencode.encode
File "rencode/rencode.pyx", line 247, in rencode._rencode.encode_list
File "rencode/rencode.pyx", line 320, in rencode._rencode.encode
Exception: type <class 'memoryview'> not handled
2022-11-04 15:57:47,403 unsupported type: <class 'memoryview'> in 'draw' packet->[7]
2022-11-04 15:57:47,403 Error: error in network packet write/format
2022-11-04 15:57:47,403 type <class 'memoryview'> not handled
Traceback (most recent call last):
File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 404, in write_format_thread_loop
self._add_packet_to_queue(*gpc())
File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 421, in _add_packet_to_queue
chunks = self.encode(packet)
File "/usr/lib/python3/dist-packages/xpra/net/protocol.py", line 649, in encode
main_packet, proto_flags = self._encoder(packet)
File "/usr/lib/python3/dist-packages/xpra/net/packet_encoding.py", line 36, in do_rencode
return rencode_dumps(v), FLAGS_RENCODE
File "rencode/rencode.pyx", line 334, in rencode._rencode.dumps
File "rencode/rencode.pyx", line 314, in rencode._rencode.encode
File "rencode/rencode.pyx", line 247, in rencode._rencode.encode_list
File "rencode/rencode.pyx", line 320, in rencode._rencode.encode
Exception: type <class 'memoryview'> not handled
2022-11-04 15:57:47,404 Python/GTK3 client 1 disconnected.
2022-11-04 15:57:47,418 Last client has disconnected, terminating
2022-11-04 15:57:47,418 xpra X11 seamless server is terminating
2022-11-04 15:57:47,431 waiting for initialization thread to complete
2022-11-04 15:57:47,445 OpenGL is supported on display ':0'
2022-11-04 15:57:47,445 using 'llvmpipe (LLVM 7.0, 256 bits)' renderer
2022-11-04 15:57:47,445 automatic picture encoding enabled, also available:
2022-11-04 15:57:47,445 h264, vp9, vp8, png, png/P, png/L, webp, rgb24, rgb32, jpeg, jpega, scroll
2022-11-04 15:57:47,452 removing unix domain socket '/run/user/2029/xpra/xpra1-0'
2022-11-04 15:57:47,452 removing unix domain socket '/home/user/.xpra/xpra1-0'
2022-11-04 15:57:47,452 removing unix domain socket '/run/user/2029/xpra/0/socket'
/usr/lib/python3/dist-packages/gi/overrides/Gtk.py:1641: Warning: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
return _Gtk_main(*args, **kwargs)
2022-11-04 15:57:47,754 killing xvfb with pid 5700
[5745:5812:1104/155747.754513:FATAL:bus.cc(1226)] D-Bus connection was disconnected. Aborting.
[5826:5826:1104/155747.755329:ERROR:connection.cc(46)] X connection error received.
Gdk-Message: 15:57:47.757: Xpra: Fatal IO error 0 (Erfolg) on X server :0.

[5826:5859:1104/155747.756472:ERROR:connection.cc(46)] X connection error received.
[1104/155747.782998:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq: No such file or directory (2)
[1104/155747.783583:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq: No such file or directory (2)
Gdk-Message: 15:57:47.786: google-chrome: Fatal IO error 11 (Die Ressource ist zur Zeit nicht verfügbar) on X server :0.
```

**Comment by totaam** [comment]
Got it!
Now I'm not sure why your setup ends up using the legacy `rencode` encoder instead of `rencodeplus` (that's another problem) - I believe the compressed are unwrapped and inlined *after* the `memoryview` safety check.

@badmin-c Can you try applying c8aea5aa35825dfe0a1ba5a289a7b488f837fdd7 ?
I believe the problem should also go away if you run with `--env=XPRA_INLINE_SIZE=0`

**Comment by totaam** [comment]
@badmin-c [xpra 4.4.2](https://github.com/Xpra-org/xpra/releases/tag/v4.4.2) includes this fix - the repositories are being updated.
I really hope this was the correct fix...

Note that when I tested, my systems were using the correct `rencodeplus` packet encoder - so there is something fishy about your setup that makes it use the legacy `rencode` one instead.

**Comment by totaam** [comment]
@badmin-c I suspect that something must be installed differently on that client.
It could be anything, OTOH:
* anti-virus software sometimes decides to quarantine random files
* conflicts with another DLL found on the `PATH` preventing `rencodeplus` from loading
* a stray configuration file override, `Xpra_cmd.exe showsetting encoders` would show it
etc.
`Network_info.exe` should be showing `rencodeplus` as available on all clients that run 4.4.x

**Comment by badmin-c** [comment]
> * anti-virus software sometimes decides to quarantine random files

All our (windows) systems use Trend Micro Security Agent, globally configured so settings should be the same on every client

> * conflicts with another DLL found on the `PATH` preventing `rencodeplus` from loading

%PATH% from my (normally functional) client:
```
echo %PATH%
C:\Oracle\product\19.0.0\client_1\bin;C:\ORA11g\Oracle11R2\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\Common Files\FFUMS\;C:\Program Files (x86)\ZANTAZ\EAS Outlook Addin\;C:\Saperion\;C:\Users\vogfra\AppData\Local\Microsoft\WindowsApps;
```
%PATH% from the client in question:
```
echo %PATH%

C:\Program Files\AdoptOpenJDK\jre-11.0.11.9-hotspot\bin;C:\ORA11g\Oracle11R2\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\ZANTAZ\EAS Outlook Addin\;C:\Program Files (x86)\Pulse Secure\VC142.CRT\X64\;C:\Program Files (x86)\Pulse Secure\VC142.CRT\X86\;C:\Program Files (x86)\dotnet\;C:\Program Files\dotnet\;C:\Program Files (x86)\Common Files\Pulse Secure\TNC Client Plugin\;C:\Program Files (x86)\Common Files\Pulse Secure\VC142.CRT\X64\;C:\Program Files (x86)\Common Files\Pulse Secure\VC142.CRT\X86\;C:\Users\freuwe\AppData\Local\Microsoft\WindowsApps;
```

 
> * a stray configuration file override, `Xpra_cmd.exe showsetting encoders` would show it
That command doesn't seem to work... Kinda? Output from my client:
```
Xpra_cmd.exe showsetting encoders
'encoders' is not a valid setting
* default config:
* 'C:\\Users\\vogfra\\Desktop\\Xpra-Python3-x86_64_4.4.2-r0\\etc\\xpra':
* 'C:\\Users\\vogfra\\AppData\\Roaming\\Xpra':
```
and the other one:
```
Xpra_cmd.exe showsetting encoders
Warning: invalid compressor(s) specified: lzo
'encoders' is not a valid setting
* default config:
* 'C:\\Test\\XPRA\\Xpra-Python3-x86_64_4.4.2-r0\\etc\\xpra':
* 'C:\\ProgramData\\Xpra':
* 'C:\\Users\\freuwe\\AppData\\Roaming\\Xpra':
```


>   `Network_info.exe` should be showing `rencodeplus` as available on all clients that run 4.4.x
It does, on both clients:



> * anti-virus software sometimes decides to quarantine random files

All our (windows) systems use Trend Micro Security Agent, globally configured so settings should be the same on every client

> * conflicts with another DLL found on the `PATH` preventing `rencodeplus` from loading

%PATH% from my (normally functional) client:
```
echo %PATH%
C:\Oracle\product\19.0.0\client_1\bin;C:\ORA11g\Oracle11R2\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\Common Files\FFUMS\;C:\Program Files (x86)\ZANTAZ\EAS Outlook Addin\;C:\Saperion\;C:\Users\vogfra\AppData\Local\Microsoft\WindowsApps;
```
%PATH% from the client in question:
```
echo %PATH%

C:\Program Files\AdoptOpenJDK\jre-11.0.11.9-hotspot\bin;C:\ORA11g\Oracle11R2\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\ZANTAZ\EAS Outlook Addin\;C:\Program Files (x86)\Pulse Secure\VC142.CRT\X64\;C:\Program Files (x86)\Pulse Secure\VC142.CRT\X86\;C:\Program Files (x86)\dotnet\;C:\Program Files\dotnet\;C:\Program Files (x86)\Common Files\Pulse Secure\TNC Client Plugin\;C:\Program Files (x86)\Common Files\Pulse Secure\VC142.CRT\X64\;C:\Program Files (x86)\Common Files\Pulse Secure\VC142.CRT\X86\;C:\Users\freuwe\AppData\Local\Microsoft\WindowsApps;
```

 
> * a stray configuration file override, `Xpra_cmd.exe showsetting encoders` would show it
That command doesn't seem to work... Kinda? Output from my client:
```
Xpra_cmd.exe showsetting encoders
'encoders' is not a valid setting
* default config:
* 'C:\\Users\\vogfra\\Desktop\\Xpra-Python3-x86_64_4.4.2-r0\\etc\\xpra':
* 'C:\\Users\\vogfra\\AppData\\Roaming\\Xpra':
```
and the other one:
```
Xpra_cmd.exe showsetting encoders
Warning: invalid compressor(s) specified: lzo
'encoders' is not a valid setting
* default config:
* 'C:\\Test\\XPRA\\Xpra-Python3-x86_64_4.4.2-r0\\etc\\xpra':
* 'C:\\ProgramData\\Xpra':
* 'C:\\Users\\freuwe\\AppData\\Roaming\\Xpra':
```

>   `Network_info.exe` should be showing `rencodeplus` as available on all clients that run 4.4.x
It does, on both clients. 
* encoders                        : rencode, bencode, yaml, rencodeplus, none
* rencode                         : True
  - version                       : Cython, 1.0.6
* rencodeplus                     : True
  - version                       : Cython, 1.0.8

**Comment by badmin-c** [comment]
I do not have access to the system at the moment, but will check asap and report here
